app.config.js

```diff
@@ -280,6 +280,6 @@ export default {
       palette,
     },
     ...app,
-    ...eas.build.preview.env,
+    ...eas.build.development.env,
```

Comment:
Point to `preview` env as this build will be deployed on Preview environment

Reply:
Resolved.